Source/ACE.Entity/Enum/IdentifyResponseFlags.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace ACE.Network.Enum
+{
+    [Flags]
+    public enum IdentifyResponseFlags
```

Comment:
Is the an entity or network enum?  I agree most of these should be in entity.  I'm not 100% sure all of them belong in there.